hw/ip_templates/ac_range_check/dv/ac_range_check_sim.core.tpl

```diff
@@ -0,0 +1,31 @@
+CAPI=2:
+# Copyright lowRISC contributors (OpenTitan project).
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+name: ${instance_vlnv("lowrisc:dv:ac_range_check_sim:0.1")}
```

Comment:
${instance_vlnv(f"lowrisc:dv:{module_instance_name}_sim:0.1")}

Note, this corefile tempalte will get renamed to `${module_instance_name}_sim.core`